cmd/repo.go

```diff
@@ -101,6 +101,27 @@ func findTemplateURL(projectVersion ProjectVersion, templateName string) string
 	return ""
 }
 
+type indexError struct {
+	indexName string
+	theError  error
+}
+
+type indexErrors struct {
+	listOfErrors []indexError
+}
+
+func (e indexError) Error() string {
+	return "- Repository: " + e.indexName + "\n  Reason: " + fmt.Sprintf("%s", e.theError)
```

Comment:
The `fmt.Sprintf()` is unnecessary here, you should just use `e.theError.Error()`

Reply:
Done